erigon-lib/state/aggregator_files.go

```diff
@@ -35,7 +35,7 @@ type SelectedStaticFilesV3 struct {
 	dHist [kv.DomainLen][]*filesItem
 	dIdx  [kv.DomainLen][]*filesItem
 	dI    [kv.DomainLen]int
-	ii    [kv.StandaloneIdxLen]*filesItemI
+	ii    [kv.StandaloneIdxLen]filesItemI
```

Comment:
seems we never use `filesItemI.i` field. let's remove `filesItemI` class and use `*filesItem*`?